python/ray/train/_internal/session.py

```diff
@@ -74,13 +74,14 @@ class TrialInfo:
 
 
 @dataclass
-class TrainingResult:
+class _InternalTrainingResult:
```

Comment:
this one's to be deleted soon. Maybe `_LegacyTrainingResult` so we can find all the `Legacy` things